offline_verifying.md

```diff
@@ -0,0 +1,70 @@
+# Offline Verifier
+
+## Problem
+
+Currently, after a document has been issued in the Document Creator, the user has to download the file before clicking the Verify Documents tab and uploading the document for verification which may be cumbersome. Created documents can be easily accessed through their `links.self.href` attribute only because the Document creator uses this function to upload the document to a server:
+
+```js
+export const uploadToStorage = async (
+  doc: WrappedDocument,
+  documentStorage: DocumentStorage
+): Promise<AxiosResponse> => {
+  const qrCodeObj = decodeQrCode(doc.rawDocument.links.self.href);
+  const uri = qrCodeObj.payload.uri;
+
+  return axios({
```

Comment:
why is pushing the doc to storage needed ? For preview ?

Reply:
Ohh this is for the QRcode verification of a document! The document creator automatically does this for us so that when we click a link we can easily access the document and send it to the verifier.